helm/odigos/templates/cleanup/cleanup-role.yaml

```
@@ -12,8 +12,11 @@ rules:
 # For more context, see the implementation in /cli/cmd/uninstall.go.
 - apiGroups: [""]
   resources: ["configmaps"]
-  resourceNames: ["odigos-deployment"]
+  resourceNames: ["odigos-deployment", "odigos-configuration"]
   verbs: ["get"]
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["list"]
```

Need to list other CMs?
If not, please unify both entries :)
```suggestion
verbs: ["get", "list"]
```